WordPress/Classes/Services/BlogService.m

```diff
@@ -288,31 +295,68 @@ - (void)syncBlog:(Blog *)blog
 
 - (void)syncBlogStaggeringRequests:(Blog *)blog
 {
-    __weak __typeof(self) weakSelf = self;
+    [self staggerSyncPostsForBlog:blog];
+}
 
+- (void)staggerSyncPostsForBlog:(Blog *)blog
+{
     PostService *postService = [[PostService alloc] initWithManagedObjectContext:self.managedObjectContext];
+    [postService syncPostsOfType:PostServiceTypePost forBlog:blog success:^{
+        [self staggerSyncPagesForBlog:blog];
```

Comment:
Just wondering: synch post triggers synch pages which triggers synch comments.  Is this chain always desired?